packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.controller.js

```diff
@@ -0,0 +1,88 @@
+import { COMFORT_EXCHANGE_TYPE_ERROR } from './pack-xdsl-access-comfort-exchange.constant';
+
+export default class XdslAccessComfortExchangeCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    OvhApiXdsl,
+    TucToast,
+    TucToastError,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OvhApiXdsl = OvhApiXdsl;
+    this.TucToast = TucToast;
+    this.TucToastError = TucToastError;
+  }
+
+  $onInit() {
+    this.exchange = {
+      order: {
+        url: null,
+        id: null,
+        priceWithoutTax: null,
+        priceWithTax: null,
+        tax: null,
+      },
+      isSuccess: false,
+    };
+    this.isAvailable = false;
+    this.getListOpenedRMA();
+  }
+
+  getListOpenedRMA() {
```

Comment:
IMO, it should be in components bindings and injected via state resolves.

Reply:
Why was this resolved ? It's not resolved

Reply:
I'm not agree with this remark. This method is called only if there is opened RMA, not every time we're going on the RMA page. So I didn't do in the resolve.